inference/enterprise/parallel/README.md

```diff
@@ -0,0 +1,32 @@
+# Parallel Robofolow Inference server
+
+Introducing the highly concurrent implementation of Roboflow Inference Server! This version of the server accepts and processes requests asynchronously, running the web server, preprocessing, auto batching, inference, and post processing all in separate threads to increase server fps throughput. Separate requests to the same model will be batched on the fly as allowed by `$MAX_BATCH_SIZE`, and then response handling will occurr independently. Images are passed via python's SharedMemory module to maximize throughput.
+
+> ⚠️ Currently, only Object Detection, Instance Segmentation, and Classification models are supported by this module. Core models are not enabled
+
+> ⚠️ We require a Roboflow Enterprise License to use this in production. See LICENSE.txt for details
+
+## How Do I use it?
+You can build the server using `./inference/enterprise/parallel/build.sh` and run it using `./inference/enterprise/parallel/run.sh`
+
+We provide a container at dockerhub that you can pull using `docker pull roboflow/roboflow-inference-server-gpu-parallel:latest`. If you are pulling a pinned tag, be sure to change the `$TAG` variable in `run.sh`.
+
+This is a drop in replacement for the old server, so you can send requests using the [same API calls](https://inference.roboflow.com/quickstart/http_inference/#step-2-run-inference) you were using previously
+
+
+## Performance
+We measure and report performance across a variety of different task types, by selecting random models found on Roboflow Universe.
+
+### Methodology
+
+The following metrics are taken on a machine with 8 cores and 1 T4 gpu. The fps metrics reflect best out of 3 trials.
```

Comment:
"1" should be "one". "3" should be "three"

Numbers below 10 should be written out (i.e. "one", "two", "three").